renderer/sources.gni

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2020 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+brave_chrome_renderer_sources = [
+  "//brave/renderer/brave_content_renderer_client.cc",
+  "//brave/renderer/brave_content_renderer_client.h",
+]
+brave_chrome_renderer_public_deps = [
+  "//brave/components/content_settings/renderer",
```

Comment:
this dep is actually not correct, it was probably leftover from some previous change

Reply:
yes, I tried to remove the dep though, there is a use of content_settings::BraveContentSettingsAgentImpl in //chrome/renderer/chrome_content_renderer_client.cc - https://github.com/brave/brave-core/blob/master/patches/chrome-renderer-chrome_content_renderer_client.cc.patch#L10.